fix for wrong user:owner rights

Joerg Bornkessel <hd_brummy@gentoo.org>
Sep 18 2006
diff -Naur vdrconvert.orig/bin/vdrconvert.sh vdrconvert/bin/vdrconvert.sh
--- vdrconvert.orig/bin/vdrconvert.sh	2006-09-18 12:46:33.584202505 +0200
+++ vdrconvert/bin/vdrconvert.sh	2006-09-18 12:48:24.759150507 +0200
@@ -137,7 +137,7 @@
 
  [ -d $TEMPDIR ] 	|| mkdir -p $TEMPDIR
  [ -d $LOGDIR ] 	|| mkdir -p $LOGDIR
- [ -d $VCOQUEUEDIR ] 	|| mkdir -p $VCOQUEUEDIR
+ [ -d $VCOQUEUEDIR ] 	|| mkdir -p $VCOQUEUEDIR && chown vdr:vdr $VCOQUEUEDIR -R
  [ -d $TEMPDIR ] 	|| mkdir -p $TEMPDIR
  [ -d $DESTDIR ] 	|| mkdir -p $DESTDIR
  
@@ -148,6 +148,7 @@
     if [ ! -f $VCOQUEUEDIR/$JOBFIL ]
     then 
 	touch $VCOQUEUEDIR/$JOBFIL
+	chown vdr:vdr $VCOQUEUEDIR/$JOBFIL
     fi
 
     let LINES=0
